app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -26,36 +31,57 @@ const Form = ({ onClose, refetch, note, isEdit }) => {
   return (
     <Formik
       initialValues={note}
-      validationSchema={NOTES_FORM_VALIDATION_SCHEMA}
+      validationSchema={VALIDATION_SCHEMA}
       onSubmit={handleSubmit}
     >
       {({ isSubmitting }) => (
-        <FormikForm className="w-full">
+        <FormikForm noValidate className="w-full">
```

Comment:
What's the need for this `noValidate` class here?

Reply:
Without noValidate the browser validations are shown before the formik's validations
<img width="382" alt="image" src="https://github.com/shreyakurian02/neeto-ui-challenge/assets/63491645/d7037735-0c61-4a53-8c7c-908f1050edaf">

Reply:
Got it. Thanks for clarifying.